tests/test_zzz_wrapper_cache.py

```diff
@@ -0,0 +1,64 @@
+import gc
+
+from delb import is_tag_node, is_text_node, tag, Document
+from _delb.nodes import NodeBase, TagNode, _wrapper_cache
+
+
+# the test order in this module matters
+
+
+def test_document_root_sustains(long_term_references):
+    document = long_term_references.pop("a_document")
+    gc.collect()
```

Comment:
```suggestion
    gc.collect()
    document = long_term_references.pop("a_document")

```

Reply:
tbh I forgot what switching these two was supposed to achieve...

Reply:
this order proves that the document instance survives a gc. the other order didn't make sense regarding the test question. (the expected gc runs were and are implicitly executed anyway.)

Reply:
alright